test/Performance/HelloWorld.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Would it then make sense to break the different operations into different files, or is the focus of this file purely how performant different operations are against a hello world app?